frontend/src/scenes/insights/LineGraph/lineGraphLogic.ts

```diff
@@ -0,0 +1,54 @@
+import { kea } from 'kea'
+import { TooltipItem } from 'chart.js'
+import { GraphDataset } from '~/types'
+import { SeriesDatum } from 'scenes/insights/InsightTooltip/insightTooltipUtils'
+import { lineGraphLogicType } from './lineGraphLogicType'
+
+export const lineGraphLogic = kea<lineGraphLogicType>({
+    path: ['scenes', 'insights', 'LineGraph', 'lineGraphLogic'],
+    selectors: {
+        createTooltipData: [
+            () => [],
+            () =>
+                (tooltipDataPoints: TooltipItem<any>[], filterFn: (s: SeriesDatum) => boolean): SeriesDatum[] => {
```

Comment:
I don't quite see why this is a Kea logic instead of just a standalone `createTooltipData` util?

Reply:
There's no real reason, but I wanted to start moving away state from the linegraph component and into its own separate logic. This is my attempt at chipping away at that tech debt, but perhaps a todo comment here will also encourage people to add things here too